src/os/CMakeLists.txt

```diff
@@ -115,16 +120,22 @@ endif()
 
 if(WITH_SPDK)
   include_directories("${CMAKE_SOURCE_DIR}/src/spdk/include")
+  add_custom_target(build_dpdk
```

Comment:
@tchaikov Thanks. it is a good question, and I also consider it:
there are two dpdks in ceph now:
1\) ceph/src/dpdk, which is a submodule of ceph. This dpdk is used by async messenger.
2\) ceph/src/spdk/dpdk, which is a submodule of spdk, and cloned with spdk repo by 
git submodule update --init --**recursive**

I think it is ok for spdk submodule to use the dpdk introduced by itself, it has the matched version with spdk submodule. 

In addition, if we consider another dpdk already installed, should we consider the spdk already installed instead of spdk submodule?